internal/command/docker.go

```diff
@@ -38,6 +62,34 @@ type Options struct {
 	Debug        bool     // Debug if true enable log verbosity
 }
 
+// initIsPodman returns true if docker is an alias to podman (e.g. via podman-docker).
+func initIsPodman() bool {
```

Comment:
What about to add a function that returns the `engine` in use, and optionally some helper isXXX  functions ? 
Example:
`func engine() (string, error)` where string can be `docker` or `podman`
`func isEngineDocker() bool` 
`func isEnginePodman() bool`
 
This will allow to remove the init function and add support for new engines.

Reply:
That's a good idea, I will make the changes as soon as I can. I'm back from vacations, I left the PC aside for 3 weeks, sorry :)

Reply:
Done in  https://github.com/fyne-io/fyne-cross/pull/42/commits/8edf19f844b25e932571c4975900d478ed72a555